app/models/osbb.rb

```diff
@@ -5,6 +5,7 @@ class Osbb < ApplicationRecord
   validates :name, presence: true, length: { minimum: 3, maximum: 255 }
   validates :phone, format: { with: VALID_PHONE }
   validates :email, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }
+  has_many :users, dependent: :nullify
```

Comment:
change association name to members + add an association for lead member

Reply:
ok